### ISSUE TITLE
Does not work with specific input file

### ISSUE THREAD

**Issue by rolandstarke** [TNOCS/minify-geojson#23]
Hey, 

I am new to geojson so i hope its not an obvious mistake.

Downloading the file: https://github.com/isellsoap/deutschlandGeoJSON/blob/master/2_bundeslaender/4_niedrig.geojson

and running 

```bash
minify-geojson  -c 5 .\4_niedrig.geojson
```

gives the following error and no output file is generated.

```
SyntaxError: Unexpected token , in JSON at position 0
    at JSON.parse (<anonymous>)
    at OgrJsonStream._transform (node_modules\minify-geojson\node_modules\ogr-json-stream\index.js:57:25)
    at OgrJsonStream.Transform._read (node_modules\minify-geojson\node_modules\readable-stream\lib\_stream_transform.js:184:10)
    at OgrJsonStream.Transform._write (node_modules\minify-geojson\node_modules\readable-stream\lib\_stream_transform.js:172:83)
    at doWrite (node_modules\minify-geojson\node_modules\readable-stream\lib\_stream_writable.js:428:64)
    at writeOrBuffer (node_modules\minify-geojson\node_modules\readable-stream\lib\_stream_writable.js:417:5)
    at OgrJsonStream.Writable.write (node_modules\minify-geojson\node_modules\readable-stream\lib\_stream_writable.js:334:11)
    at ReadStream.ondata (_stream_readable.js:713:22)
    at ReadStream.emit (events.js:200:13)
    at addChunk (_stream_readable.js:294:12)
```

**Comment by erikvullings** [comment]
Since you're new to geojson, are you sure you're dealing with an actual
geojson file? The error you received suggests a formatting error in the
file, perhaps because it's not a geojson. Could you paste a part of the
file as a snippet, or try to open it in https://geojson.io...

Sorry, I've answered you to quickly. I'll try to have a look at your file today...

**Comment by rolandstarke** [comment]
OK, thanks, (the file is fine on geojson.io).

Here is the file I am using now (which gives the error):

```json
{
    "type": "FeatureCollection",
    "features": [
        {
            "type": "Feature",
            "id": 0,
            "properties": {
                "ID_0": 86,
                "ISO": "DEU",
                "NAME_0": "Germany",
                "ID_1": 1,
                "NAME_1": "Baden-Württemberg",
                "NL_NAME_1": null,
                "VARNAME_1": null,
                "TYPE_1": "Land",
                "ENGTYPE_1": "State"
            },
            "geometry": {
                "type": "Polygon",
                "coordinates": [
                    
                        [
                            [
                                8.708373069763297,
                                47.715557098388672
                            ],
                            [
                                8.722451210022371,
                                47.696529388427791
                            ],
                            [
                                8.661308288574162,
                                47.695213317871264
                            ],
                            [
                                8.708373069763297,
                                47.715557098388672
                            ]
                        ]
                    
                ]
            }
        }
    ]
}
```

OS: windows, line endings of the file: LF

```bash
node --version
#v12.4.0
npm show minify-geojson version
#1.2.6
```

**Comment by erikvullings** [comment]
@rolandstarke Thanks for noting the issue. I've resolved it for now (and tested that it works on node v10.16 for both your files - the file with one feature, and the 4_niedrig version), but am not completely happy with the solution. When the streaming mode fails (which will always fail for files with only one feature), I use the regular mode, reading the whole input geojson into memory.  Have to delve deeper into the issue...
For now, you can just grab the latest patch-release version from npm (v1.2.7)

**Comment by erikvullings** [comment]
@rolandstarke I've just replaced the offending library (ogr-json-schema) with JSONstream, and now it works as it should again, in full streaming mode. Released as v1.3.0.
